src/firmware/boards/legacy_robot_stm32f4/io/wheels.c

```diff
@@ -43,6 +43,16 @@
 
 #define SWITCH_RESISTANCE 0.6f  // ohms—L6234 datasheet
 
+// Some of these are duplicated from 2015_wheel_constants.h since using those values
+// directly is too much effort for this legacy robot and build system
```

Comment:
:confused: - why is this too much effort?

Reply:
actually it's not that bad...